docs/docsite/rst/tips_tricks/sample_setup.rst

```diff
@@ -191,7 +191,7 @@ Ansible loads any file called ``main.yml`` in a role sub-directory. This sample
       service:
         name: ntpd
         state: started
-        enabled: yes
+        enabled: true
```

Comment:
Since you are here, can you please add FQCN to all the modules' examples? I am OK doing this in the same PR or a separate PR.

Reply:
Hey! @Akasurde since I am new to all this I am not very used to technical terms here but I am eager to learn .

Basically you want me to replace `postgresql_user` with `community.postgresql.postgresql_user` right ?

Reply:
Yes, you are right. Ansible now uses Fully Qualified Collection Names (FQCN) to refer to the modules. So `service` will be referred to as `ansible.builtin.service`. You can check this by using the `ansible-doc` command - 

```
# ansible-doc service
...
> ANSIBLE.BUILTIN.SERVICE    (/src/ansible/lib/ansible/modules/service.py)
...
```